packages/kbn-unified-data-table/src/components/data_table_additional_display_settings.test.tsx

```diff
@@ -176,6 +177,45 @@ describe('UnifiedDataTableAdditionalDisplaySettings', function () {
         findTestSubject(component, 'unifiedDataTableSampleSizeInput').last().prop('value')
       ).toBe(validIntegerValue);
     });
+
+    it('should not fail if sample size is not step of 10', async () => {
```

Comment:
`Nit`: Consider adding a test case for sample size for less than `RANGE_MIN_SAMPLE_SIZE` since the value of step changes accordingly.

Reply:
Added via https://github.com/elastic/kibana/pull/192152/commits/5af01879e93b7370ee98b8cba0a7589b776971e0

Reply:
Thank you.